airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -204,8 +224,12 @@ def __init__(
         self.wait_for_completion = wait_for_completion
         self.cluster_id = cluster_id
         self.aws_conn_id = aws_conn_id
-        self.waiter_countdown = waiter_countdown
-        self.waiter_check_interval_seconds = waiter_check_interval_seconds
+        self.waiter_max_attempts = (
+            (waiter_countdown // waiter_check_interval_seconds)
+            if waiter_max_attempts is NOTSET
+            else waiter_max_attempts
+        )
+        self.waiter_delay = waiter_check_interval_seconds if waiter_delay is NOTSET else NOTSET
```

Comment:
Maybe I'm confused, but don't you want this?:
 ```suggestion
         self.waiter_delay = waiter_check_interval_seconds if waiter_delay is NOTSET else waiter_delay
```

Reply:
100% my mistake.   Thanks for the catch.